yaksh/templates/yaksh/courses.html

```diff
@@ -151,9 +151,14 @@
                     {% for unit in module.get_learning_units %}
                         <ul class="inputs-list">
                         <li>
-                        {% if unit.type == "quiz" %}
-                            <a href="{{URL_ROOT}}/exam/manage/addquiz/{{unit.quiz.id}}/{{course.id}}">
-                            {{unit.quiz.description}}</a>
+                        {% if unit.learning_type == "quiz" %}
+                            {% if unit.quiz.is_exercise %}
```

Comment:
You need to add this condition below in allotted courses. Since teacher can also view courses. Currently allotted courses and created courses are displayed separately.

Reply:
Added